lib/screens/home/profile/profile_page.dart

```diff
@@ -51,7 +53,12 @@ class _ProfilePageState extends State<ProfilePage> {
                   ),
                   FlatButton(
                     child: Text("Add pet"),
-                    onPressed: () {},
+                    onPressed: () {
+                      Navigator.of(context).pushNamed(
+                              AddPetPage.routeName,
+                              arguments: Pet.empty()
```

Comment:
There's no need for this argument to be passed.

Reply:
The page and the form for add-pet will be used for edit-pet also. And, in case of editing, a pet object with all its attributes will be required as argument. So, I'll not change this, even if in this case it's only an 'empty' pet object.